Compile Error for px4-dev Mac OS Mojave

I got the following error when I typed 

`brew install px4-dev`

The error message: 
```
==> Installing px4-dev from px4/px4
==> Installing dependencies for px4/px4/px4-dev: fastrtps and fastcdr
==> Installing px4/px4/px4-dev dependency: fastrtps
==> Downloading http://px4-tools.s3.amazonaws.com/fastrtps-1.5.0.mojave.bottle.tar.gz

curl: (22) The requested URL returned error: 403 Forbidden
Error: Failed to download resource "fastrtps"
Download failed: http://px4-tools.s3.amazonaws.com/fastrtps-1.5.0.mojave.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Downloading http://px4-tools.s3.amazonaws.com/fastrtps-v1.5.0p.tar.gz
Already downloaded: /Users/xxv/Library/Caches/Homebrew/downloads/c137e38e8520bbfa08069b90bbe92248e54b923a847a2a3b3290230451b82201--fastrtps-v1.5.0p.tar.gz
==> Patching
patching file CMakeLists.txt
patch unexpectedly ends in middle of line
Hunk #1 succeeded at 373 with fuzz 1.
==> cmake -DTHIRDPARTY=ON -DBUILD_JAVA=ON -DCMAKE_INSTALL_PREFIX:PATH=/usr/local/Cellar/fastrtps/1.5.0 ..
==> make
Last 15 lines from /Users/xxv/Library/Logs/Homebrew/fastrtps/02.make:
[ 86%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/rtps/builtin/data/WriterProxyData.cpp.o
[ 87%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/rtps/builtin/data/ReaderProxyData.cpp.o
[ 89%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/rtps/flowcontrol/ThroughputController.cpp.o
[ 90%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/rtps/flowcontrol/ThroughputControllerDescriptor.cpp.o
[ 91%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/rtps/flowcontrol/FlowController.cpp.o
[ 92%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/rtps/exceptions/Exception.cpp.o
[ 93%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/rtps/attributes/PropertyPolicy.cpp.o
[ 94%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/rtps/common/Token.cpp.o
[ 95%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/xmlparser/XMLParserCommon.cpp.o
[ 96%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/xmlparser/XMLProfileElementParser.cpp.o
[ 97%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/xmlparser/XMLEndpointParser.cpp.o
[ 98%] Building CXX object src/cpp/CMakeFiles/fastrtps.dir/xmlparser/XMLProfileParser.cpp.o
[100%] Linking CXX shared library libfastrtps.dylib
[100%] Built target fastrtps
make: *** [all] Error 2
```

--- a/fastcdr.rb
+++ b/fastcdr.rb
@@ -4,8 +4,8 @@ class Fastcdr < Formula
   homepage "http://www.eprosima.com/index.php/products-all/eprosima-fast-rtps"
   version "1.6.0"
   desc "OMG RTPS / DDS implementation."
-  url "https://github.com/eProsima/Fast-CDR/archive/v1.0.6.tar.gz"
-  sha256 "ce9ee04a7b242721df3a0af98e6ae4f8afe9d536e7d59405322e03a6d251d471"
+  url "https://github.com/eProsima/Fast-CDR/archive/v1.0.8.tar.gz"
+  sha256 "d47d924b76bc2fc7821dcadda4f8d1cb0f56df8353aa57cdf1106c49a612507f"
 
   depends_on "cmake" => :build
 

